src/routes/index.routes.js

```diff
@@ -2,13 +2,15 @@ import express from 'express';
 // eslint-disable-next-line import/order
 import { protect } from '@/middleware/auth';
 import authRouter from './auth.routes';
+import labRouter from './lab.routes';
 import submissionRouter from './submission.routes';
 import userRouter from './user.routes';
 
 const router = express.Router();
 
 router.use('/auth', authRouter);
 router.use('/users', protect, userRouter);
+router.use('/labs', protect, labRouter);
```

Comment:
Control access via authorizer middleware

Steps to follow,
- import 'authorizer' middleware
- call it on labs rout,
```router.use('/labs', protect,authorizer(['ADMIN', 'LECTURER']), labRouter); ```